src/wallet/rpcwallet.cpp

```diff
@@ -181,6 +181,7 @@ static void WalletTxToJSON(interfaces::Chain& chain, const CWalletTx& wtx, UniVa
             rbfStatus = "yes";
     }
     entry.pushKV("bip125-replaceable", rbfStatus);
+    entry.pushKV("inmempool", wtx.fInMempool);
```

Comment:
Probably should be omitted entirely if `confirms != 0`

Reply:
I think so too.